layouts/blog_article.tpl

```diff
@@ -1,5 +1,7 @@
 <!DOCTYPE html>
 {% include "template-variables" %}
+{% include "blog-article-variables" %}
+{% include "blog-comment-form-variables" %}
```

Comment:
You shouldn't `include` **"blog-comment-form-variables"** here. Instead it should be `include`-d here: https://github.com/Voog/design-sapporo/blob/master/components/blog-article-comments.tpl#L6